src/content/en/2020/markup.md

```diff
@@ -543,58 +608,63 @@ When we built our tests we didn’t check for the presence of images, but only l
 | … | | |
 | SVG | 5,328 | 0.08% |
 
-<figcaption>{{ figure_link(caption="Common favicon approaches.", sheets_gid="1930085905", sql_file="pages_almanac_by_device_and_favicon_image_type.sql") }}</figcaption>
+<figcaption>{{ figure_link(caption="Common favicon formats.", sheets_gid="1930085905", sql_file="pages_almanac_by_device_and_favicon_image_type.sql") }}</figcaption>
 </figure>
 
-There may be a couple of surprises in here:
+There are a couple of surprises in here:
 
-* Support for other formats is there but ICO is still the go-to format for favicons on the Web.
+* Support for other formats is there but ICO is still the go-to format for favicons on the web.
 * JPG is a relatively popular favicon format even though it may not yield the best results (or a comparatively large file size) for many favicon sizes.
-* WebP is—twice as popular as SVG! (This might change, however, with [SVG favicon support improving](https://caniuse.com/link-icon-svg).)
+* WebP is twice as popular as SVG! This might change, however, with [SVG favicon support](https://caniuse.com/link-icon-svg) improving.
 
 ### Button and input types
 
-There has been a lot of discussion on buttons lately and how often they are misused. We looked into this to present findings on some of the native HTML buttons:
+There has been a lot of [discussion](https://adrianroselli.com/2016/01/links-buttons-submits-and-divs-oh-hell.html) on buttons lately and how often they are misused. We looked into this to present findings on some of the native HTML buttons.
+
+{{ figure_markup(
+  caption="Percent of pages with button elements.",
+  content="60.56%",
+  classes="big-number",
+  sheets_gid="410549982",
+  sql_file="pages_markup_by_device.sql"
+) }}
 
 <figure markdown>
-| Elements | Occurrences | Page percentage |
+| Button types | Occurrences | Percentage |
```

Comment:
“Percentage” is confusing and, I believe, wrong here. I’m not sure whether “Page percentage”, as per the draft, is perfectly clear, but just “Percentage” does not work. @zcorpan, you seem to have a clearer view here, can you help out and advise on alternative wording?

Reply:
Is it percentage of pages? If so, maybe "Percentage (pages)"

Reply:
Yeah

Reply:
Is the corresponding [`sheets_gid`](https://docs.google.com/spreadsheets/d/1Ta7amoUeaL4pILhWzH-SCzMX9PsZeb1x_mwrX2C4eY8/edit#gid=410549982) correct? I'm not able to match up these results with the sheet. 15.9M can't be right because there are only 6,347,640 mobile pages in the dataset. @j9t or @Tiggerito could you clarify where these values came from?

Reply:
@catalinred, can you advise? I think this is due to [this section](https://docs.google.com/document/d/1m1YvDqWgZfqmcmAEkCnXncIJwsM1QCu6Rv7aPWjG1Ms/edit#heading=h.nm1lcnapqkhr) pulling from four sheets, and I had a similar problem tracing this down.